tests/ssg_test_suite/rule.py

```diff
@@ -55,20 +57,19 @@ def get_viable_profiles(selected_profiles, datastream, benchmark):
             if ds_profile.endswith(sel_profile):
                 valid_profiles += [ds_profile]
     if not valid_profiles:
-        logging.error('No profile matched with "{0}"'
+        logging.error('No profile ends with "{0}"'
                       .format(", ".join(selected_profiles)))
     return valid_profiles
 
 
-def _run_with_stdout_logging(command, log_file):
-    log_file.write(" ".join(command) + "\n")
+def _run_with_stdout_logging(command, args, log_file):
+    log_file.write(" ".join((command,) + args) + "\n")
```

Comment:
This function is always used with first parameter as a string:
```
if _run_with_stdout_logging("ssh", args, log_file) 
if _run_with_stdout_logging("scp", args, log_file) 
````
I think it would be more readable this way:
```
log_file.write("{0} {1}\n".format(command, " ".join(args))
```
That way we would avoid commas.